### PR TITLE
DE46605: Show LOR embedded preview skeleton on page load

### PR DESCRIPTION
**Pull request by brissons**
## Relevant Rally Links

- [DE46605](https://rally1.rallydev.com/#/289692574792/custom/355050439968?detail=%2Fdefect%2F620969023061&fdp=true): LOR Embed View - Missing loading skeleton for LOR Embed Preview

## Description

The LOR Embed Preview was missing a loading skeleton when the page loads, appearing only once the entity has been loaded. This PR makes sure that the skeleton appears while the entity is being fetched.

**Note**: The non-embedded view shares a loading skeleton with the embedded view. This is because we don't know if the preview can be embedded until the entity is fetched.

## Goal/Solution

Have the embedded preview appear when the page is loading. 

## Video/Screenshots
**Before**:

lor-preview-before

**After**:

lor-preview-after

**After (not embedded)**:

lor-preview-after-ne

## Remaining Work

- [x] Dev Testing: another developer will test this code on their machine

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-content-editor/content-file/d2l-activity-content-file-detail.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ class ContentFileDetail extends SkeletonMixin(ErrorHandlingMixin(LocalizeActivit
 					.href="${this.activityUsageHref}"
 					.token="${this.token}"
 					?skeleton=${this.skeleton}
-					?expanded=${true}
+					expanded
```
Comment (brissons) [inline components/d2l-activity-editor/d2l-activity-content-editor/content-file/d2l-activity-content-file-detail.js:184]: Linter was complaining about these.

**Review by ChrisKraljevicD2L — APPROVED** [review]
LGTM :+1:

**Review by coffeexcode — COMMENTED** [review]
LGTM

**Review by ChrisKraljevicD2L — APPROVED** [review]
dev tested, works a-ok!

**Pull request merged**